plugins/GSdx/GS.h

```diff
@@ -1425,3 +1425,25 @@ enum class Filtering : uint8
 	Trilinear_Bilinear_Forced,
 	Trilinear_Always
 };
+
+//Enum order matters! Don't change it!
+enum class CRCHackLevel : int8
+{
+	Automatic = -1,
+	None = 0,
+	Minimum = 1,
+	Partial = 2,
+	Full = 3,
+	Aggressive = 4
```

Comment:
Well, it's unlikely that strings would be changed. And if they are changed, then the user would end up on the default setting if the string doesn't match. My suggestion is out of scope for this PR though - I'd like a string not only for the CRC hack level, but also some of the other settings too (renderer comes to mind but there's probably others). It isn't truly related to this PR - we can have the enum and/or the suggestion.

So basically I have no opinion on whether to explicitly assign values to each enum or not (I also have no opinion on the switch/if/ternary) but well, I was sort of asked for an opinion and so I provided one ;)